File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/GroupFragAdm/dataModelRecVCourseSelectedFAdm.java
package com.example.adminapp.GroupFragAdm;

public class dataModelRecVCourseSelectedFAdm {
    String chapterName;
    int totalNoOfModules;

    public dataModelRecVCourseSelectedFAdm() {}  // no-args constructor

    public dataModelRecVCourseSelectedFAdm(String chapterName, int totalNoOfModules) {
        this.chapterName = chapterName;
        this.totalNoOfModules = totalNoOfModules;
    }

    public String getChapterName() {
        return chapterName;
    }

    public void setChapterName(String chapterName) {
        this.chapterName = chapterName;
    }

    public int getTotalNoOfModules() {
        return totalNoOfModules;
    }

    public void setTotalNoOfModules(int totalNoOfModules) {
        this.totalNoOfModules = totalNoOfModules;
    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/GroupFragAdm/dataModelRecVCourseSelectedFAdm.kt
package com.example.adminapp.GroupFragAdm

class dataModelRecVCourseSelectedFAdm {
    var chapterName: String? = null
    var totalNoOfModules = 0

    constructor() // no-args constructor
    constructor(chapterName: String?, totalNoOfModules: Int) {
        this.chapterName = chapterName
        this.totalNoOfModules = totalNoOfModules
    }

//    fun getChapterName() : String?{
//        return chapterName;
//    }
//
//    fun getTotalNoOfModules() : Int {
//        return totalNoOfModules;
//    }
//
//    fun setChapterName(chapterName: String?){
//        this.chapterName = chapterName;
//    }
//
//    fun setTotalNoOfModules(totalNoOfModules: Int){
//        this.totalNoOfModules = totalNoOfModules;
//    }
}
